Formula/c/certifi.rb

```diff
@@ -25,6 +24,9 @@ def pythons
   end
 
   def install
+    # Ensure uniform bottles
+    inreplace "README.rst", "/usr/local", HOMEBREW_PREFIX
+
```

Comment:
A bit trickier as the `METADATA` file (generated in venv) is fixed but it looks like the `README.rst` itself (installed via default call to `prefix.install_metafiles`) is not substituted with placeholder.